docs/getting-started/tools.en.md

```diff
@@ -0,0 +1,38 @@
+Here some handy tools are offered to complement your device's and wallets management.
+
+### Check SD Card
+
+<img src="../../img/maixpy_m5stickv/check-sd-card-125.png" align="right">
+<img src="../../img/maixpy_amigo_tft/check-sd-card-150.png" align="right">
+
+You can check if a SD card can be detected and read by your device and explore its content.
+
+<br><br><br><br>
+<br><br><br><br>
+
+### Delete Mnemonic
+
+<img src="../../img/maixpy_m5stickv/delete-mnemonic-125.png" align="right">
+<img src="../../img/maixpy_amigo_tft/delete-mnemonic-150.png" align="right">
+
+Delete any stored encrypted mnemonic, on device's internal memory or SD card.
```

Comment:
If I had an SD card plugged in, but my mnemonic was stored on the flash, would this try to wipe both places or just the SD card?

Ideally it would always try to wipe both, I think.

Reply:
This tool list all encrypted keys and allow you to wipe them one by one, independent of where it is stored (flash or SD card)